docs/configuration-reference/components/prometheus-operator.md

```diff
@@ -47,6 +47,12 @@ component "prometheus-operator" {
     node_selector = {
       "kubernetes.io/hostname" = "worker3"
     }
+    tolerations {
```

Comment:
I wonder if we could somehow improve this experience for users. E.g. being able to identify what workloads are part of the component and then do some smart mapping to worker groups, so you don't have to specify those manually.